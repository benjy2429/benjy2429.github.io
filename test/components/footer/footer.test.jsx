import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Footer from '../../../src/components/footer/footer';

const props = {
  menuLinks: 'menuLinks',
  socialLinks: 'socialLinks'
};

describe('Footer component', () => {
  it('renders correctly', () => {
    const component = shallow(<Footer {...props} />);
    expect(toJson(component)).toMatchSnapshot();
  });
});
