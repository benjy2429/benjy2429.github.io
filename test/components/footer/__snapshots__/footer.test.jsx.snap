// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders correctly 1`] = `
<footer
  className="footer"
>
  <div
    className="container"
  >
    <p
      className="footer-title"
    >
      Ben Carr
      <br />
      Software Developer
    </p>
    <div
      className="footer-grid"
    >
      <div
        className="footer-grid-left"
      >
        <Component
          links="menuLinks"
          title="Menu"
        />
      </div>
      <div
        className="footer-grid-left"
      >
        <Component
          links="socialLinks"
          title="Social"
        />
      </div>
      <div
        className="footer-grid-right"
      >
        Copyright Â© 
        twenty-eighteen
        <br />
        v1
        <br />
        <GatsbyLink
          className="link-light"
          to="/changelog"
        >
          Changelog
        </GatsbyLink>
      </div>
    </div>
  </div>
</footer>
`;
